devices/woox.js

```diff
@@ -25,6 +26,28 @@ module.exports = [
         extend: extend.light_onoff_brightness_colortemp_color({disableColorTempStartup: true}),
         meta: {applyRedFix: true},
     },
+    {
+        fingerprint: [{modelID: 'TS0201', manufacturerName: '_TZ3000_rusu2vzb'}],
+        model: 'R7048',
+        vendor: 'Woox',
+        description: 'Smart Humidity & Temperature Sensor',
+        fromZigbee: [fz.battery, fz.temperature, fz.humidity],
+        toZigbee: [],
+        exposes: [e.battery(), e.temperature(), e.humidity(), e.battery_voltage()],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            try {
+                const endpoint = device.getEndpoint(1);
+                const bindClusters = ['msTemperatureMeasurement', 'msRelativeHumidity', 'genPowerCfg'];
+                await reporting.bind(endpoint, coordinatorEndpoint, bindClusters);
+                await reporting.temperature(endpoint);
+                await reporting.humidity(endpoint);
+                await reporting.batteryVoltage(endpoint);
+                await reporting.batteryPercentageRemaining(endpoint);
+            } catch (e) {/* Not required for all: https://github.com/Koenkk/zigbee2mqtt/issues/5562 */
```

Comment:
I don't think this try/catch is needed.

Reply:
You mean the entire `configure` block? I'll go ahead & remove it then, thanks :+1:

Reply:
I don't expect the entire configure block can be removed. To test this you should factory reset the device.

Reply:
Thanks for the tip, clearly I'm not so familiar node.js/this project :see_no_evil: 

Fixed (again, and tested by resetting device & rejoining), PTAL :pray: